src/fhirConfig.ts

```diff
@@ -23,8 +23,21 @@ export interface OAuthStrategy {
     revokeUrl?: string;
```

Comment:
in the [specs](http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#using-well-known) all of the "url" fields are described as "endpoint". Example `authorizationUrl` => `authorizationEndpoint`. 

Do we want to change the interface to use the same naming spec as in `well-known`?

Pro: Easier mapping when we convert these fields into json. We just convert camelCase to snake_case.
Con: This is a breaking change and means we have to bump `interface` to version 4.

Reply:
We have to bump the version any way due to my changes -- I don't have a strong opinion it seems to be closer to the spec when using 'endpoint' so I say we do that

Reply:
sounds good, I'll change it the suffix to `endpoint`